### PR TITLE
Improve fee handling

### PR DESCRIPTION
**Pull request by chimp1984**

### CODE REVIEW

File: core/src/main/java/bisq/core/provider/mempool/TxValidator.java
Comment (ripcurlx) [inline core/src/main/java/bisq/core/provider/mempool/TxValidator.java]: Maybe call it `maybeCheckAgainstFeeFromFilter`? `maybeTestFeeFromFilter` sounds a little bit weird to me (I'm no native speaker)

Comment (chimp1984) [reply core/src/main/java/bisq/core/provider/mempool/TxValidator.java]: Done

**Review by ripcurlx — APPROVED** [review]
utACK - I only reviewed the code and tested filter creation on Regtest. For Mainnet tests I'll wait for the ACK from @jmacxx before I'll merge this PR.

**Review by ghost — APPROVED** [review]
ACK

**Pull request merged**
